meetings/2025-02-26.md

```diff
@@ -28,16 +32,73 @@
+
+### Blank nodes to be inserted in N3 Patches
+
+See [PR #711](https://github.com/solid/specification/pull/711)
+
+* HZ: There were conversations on matrix and in the PR itself.
+* ML: How can we move forward on this topic?
+* ML: Currently, we can't insert blank nodes. I understand why it is forbidden for deletion. I think for insertions, we should allow them. Ruben seems to be against; I'm not sure if he understood that we just want to allow them in insertions. Sarven approved PR. How should we move forward?
+* eP: I didn't check the minutes of last call. Is Ruben participating? Can Maxime be invited on the LWS call?
+* HZ: LWS WG is not close to that point. I don't recall Ruben in WG calls.
+* HZ: I would propose for me and Maxime to reach out to Ruben and comment and/or approve the PR.
+* JW: Ruben is at Oxford to give a talk, I can talk to him. There seems to be a misunderstanding. There is also a point, should we be proceding with this change given that N3 is not a WG spec. Should we signal possible deprication. Dorthe is discussing making N3 a standard for a while. 
+* HZ: I think the WG is nowhere near making this kind of decisions.
+* eP: useful feedback. What Sarven brought up we discussed on a few occasions. I suggested that in the end we may have to go with SPARQL updates. If the current blank node is a bug in the current spec, should we update the spec with a small update. For Solid, trying to move to SPARQL updates would be a major change, not consistent with the WG request.
+* JW: I agree. A potential phrasing would be that we recommend using SPARQL updates vs N3 patch.
+* eP: sounds reasonable. We could upen a pull request with the desired wording. I had a little chat with PAC about the maturity level. It may be worth running this by PAC - Pierre Antoine.
+* Michal: I have an impression that SPARQL Update is not a part of the specification. If it is not I would find it confusing to be recommended. I did the move from SPARQL to N3 in some of my work.
+* ML: I'm not against SPARQL-update. I think N3 patch has been introduced to resolve issues that were found in SPARQL-update (different interpretations). We may need to detail SPARQL-update more to also address those issues (so everyone use the SPARQL-update in a consistent way). 
+* JW: With Ruben I will only clarify some details and possible misunderstanding. It would be good if someone else would go throug the specification. If it is there we can create PR in with suggested note. If it is not there, we need to procede with the current PR. There is a question of raising a note to say that SPARQL Update may be reintroduced. In both cases we should cc PAC. 
+* eP: I believe SPARQL update is not in the spec. Since Ruben was working on N3 patch, I assume he's fully aware of the issue Maxime is raising. I assume Ruben should have a good recollection of the reason for dropping SPARQL update.
```

Comment:
```suggestion
* JW: With Ruben, I will only clarify some details and possible misunderstanding. It would be good if someone else would go through the specification. If it is there, we can create PR with suggested notes. If it is not there, we need to proceed with the current PR. There is a question of raising a note to say that SPARQL Update may be reintroduced. In both cases, we should cc PAC. 
* eP: I believe SPARQL update is not in the spec. Since Ruben was working on N3 Patch, I assume he's fully aware of the issue Maxime is raising. I assume Ruben should have a good recollection of the reason for dropping SPARQL Update.
```